src/walletConnect/screens/ActionRequest.tsx

```diff
@@ -93,68 +94,61 @@ function ActionRequest({ navigation, route: { params: routeParams } }: Props) {
     if (!moreInfoString) {
       return
     }
-
-    dispatch(showRequestDetails(routeParams, moreInfoString))
+    // TODO: remove this as a separate screen
```

Comment:
for now this looks okay, so can tackle in the next PR where we update the UI to be a bottom sheet